oceannavigator/frontend/src/components/OceanNavigator.jsx

```diff
@@ -340,22 +340,37 @@ function OceanNavigator(props) {
 
   return (
     <div className="OceanNavigator">
+     <div className="top-panel-components">
+  <div className="scale-viewer-container">
+    <ScaleViewer
+      dataset={dataset0}
+      mapSettings={mapSettings}
+      onUpdate={updateDataset0}
+      mapState={mapState}
+    />
+    {compareDatasets && (
       <ScaleViewer
-        dataset={dataset0}
+        dataset={dataset1}
         mapSettings={mapSettings}
         onUpdate={updateDataset0}
         mapState={mapState}
+        className="right"
       />
-      {compareDatasets ? (
-        <ScaleViewer
-          dataset={dataset1}
-          mapSettings={mapSettings}
-          onUpdate={updateDataset0}
-          mapState={mapState}
-          right={true}
-        />
-      ) : null}
-      <MinimizedPlotBar plotData={plotData} action={action} />
+    )}
+  </div>
+
+  <MinimizedPlotBar plotData={plotData} action={action} />
+  <div className="button-container">
```

Comment:
Might be able to get rid of this div now that everything is in "top-panel-components".